rpm/phonesim.spec

```diff
@@ -48,8 +49,8 @@ mkdir -p %{buildroot}%{_datadir}/applications/
 cp -a %{SOURCE2} %{buildroot}%{_datadir}/applications/
 mkdir -p %{buildroot}%{_datadir}/phonesim/
 cp -a %{SOURCE3} %{buildroot}%{_datadir}/phonesim/
-mkdir -p %{buildroot}%{_libdir}/systemd/user/
-cp -a %{SOURCE4} %{buildroot}%{_libdir}/systemd/user/
+mkdir -p %{buildroot}%{_userunitdir}
+cp -a %{SOURCE4} %{buildroot}%{_userunitdir}
 mkdir -p %{buildroot}%{_sysconfdir}/dbus-1/system.d/
```

Comment:
Can you fix that while you are at it?
It should be `%{buildroot}%{_datadir}/dbus-1/system.d/`

Similar as in polkit here:
https://github.com/sailfishos/polkit/blob/master/rpm/polkit.spec#L143

Reply:
There you go. I added also another fix I noticed now.

Reply:
Please add JB#35534 as bug ref for it.

Reply:
Ok, I will.